test/runtests.jl

```diff
@@ -24,7 +24,10 @@ const tests = [
 
 printstyled("Running tests:\n", color=:blue, bold=true)
 
-@time for test in tests
-    include("$test.jl")
-    println("\033[1m\033[32mPASSED\033[0m: $(test)")
-end
+@testset "ReadStatTables" verbose=true begin
```

Comment:
The `verbose` keyword is not in Julia v1.3? The v1.3 support could stay with the v0.2.x versions until v0.3.0 is reached for which v1.6 will be required.